### PR TITLE
docs: update self-managed-vpc-no-internet-access instructions

### PR DESCRIPTION
**Pull request by deeno35**
There are tons of new features now, update this set of instructions with a working end to end install.  Normal installs will only require what's in main.tf, but for the purpose of creating a functional end to end install, all of the addtional *_example.tf files have been added to create addtional resources necessary such as SES, Workmail, a bastion for SSH tunnel, etc.

### CODE REVIEW

File: examples/self-managed-vpc-no-internet-access/README.md [hunk 1]
```diff
@@ -1,53 +1,211 @@
...
+# Self-managed VPC with no internet access installation
 
 Some installation environments may require installing Bigeye in a VPC that has no NAT or route for
-any services or resources to public net. The method for doing this is to create your own (or bring 
-an existing) VPC that does not have internet access and pass it into the Bigeye module.  
+any services or resources to public net. The method for doing this is to create your own (or bring
+an existing) VPC that does not have internet access and pass it into the Bigeye module.
 
-This example shows the necessary configuration of variables for the Bigeye module to accomplish 
+The only file necessary is to install Bigeye with existing infrastructure (DNS domain, vpc, security groups,
```
Comment (BrianKopp) [inline examples/self-managed-vpc-no-internet-access/README.md:7]: nit: should this say `The only file necessary to ...`

File: examples/self-managed-vpc-no-internet-access/main.tf [hunk 2]
```diff
@@ -1,42 +1,113 @@
...
+  # The bastion will be used for an SSH tunnel for no internet installs where a VPN is not available.  This is here for
+  # demonstration purposes in this example, but a VPN is recommended instead for production installs.
+  bastion_enabled = false
+
+  # If you plan to use the bastion, run ssh-keygen -t rsa -b 4096 to create a key pair first if you have not already
+  bastion_ssh_public_key_file = "~/.ssh/id_rsa.pub"
+
+  # https://whatsmyip.com can be used to get your IP if you don't have it.  This ingres cidr goes into the
+  # security group allowing access to the bastion.
+  bastion_ingress_cidr = "24.7.117.81/32"
```
Comment (BrianKopp) [inline examples/self-managed-vpc-no-internet-access/main.tf:71]: Is this anonymized, or does that matter?

File: examples/self-managed-vpc-no-internet-access/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+output "bigeye_first_time_account_creation_address" {
+  value = "https://${local.vanity_alias}.${local.subdomain}/first-time"
+}
+
+output "bastion" {
+  value = one(aws_instance.bastion[*].public_dns)
+}
+
+output "bastion_ssh_user" {
+  value = "ssh ubuntu@<bastion>"
```
Comment (BrianKopp) [inline examples/self-managed-vpc-no-internet-access/outputs.tf:14]: Should we be interpolating the IP here?

**Review by BrianKopp — APPROVED** [review]

File: examples/self-managed-vpc-no-internet-access/main.tf (hunk 2 above)
Comment (deeno35) [reply examples/self-managed-vpc-no-internet-access/main.tf:71]: Yes, thanks!

File: examples/self-managed-vpc-no-internet-access/outputs.tf (hunk 3 above)
Comment (deeno35) [reply examples/self-managed-vpc-no-internet-access/outputs.tf:14]: It's one of those tricky things about condition responses in output.  I could put it in there, but then it looks strange when the flag is off for the bastion and ends up with this:

`ssh ubuntu@`

I'll see if I can improve this

**Pull request merged**